### ISSUE TITLE
Trends analysis bug - wrong conditions for selecting best RMSLE

### ISSUE THREAD

**Issue by Inglezos** [lisphilar/covid19-sir#307]
## Summary
Regarding fix #303, I found a problem inside `Trend.run()` and specifically at the RMSLE conditions that are checked to determine which dataframe to keep, either `dataframe_dict[L]` or `dataframe_dict[N]`.
In the case that the `dataframe_dict[N]` contains `inf` values, the RMSLE score is 0. Let's assume that the `dataframe_dict[L]` contains normal values and has RMSLE 0.0003; then the conditions:
```
        # Select the best dataframe
        if 0 < score_dict[L] < score_dict[N]:
            self._result_df = dataframe_dict[L]
        else:
            self._result_df = dataframe_dict[N]
```
 are **not correct** and as a result the wrong `dataframe_dict[N]` will be selected, instead of the correct one  `dataframe_dict[L]`.

I created pull request #308.

## Environment
- CovsirPhy version: 2.10.0-alpha.refactor.291
- Python version: 3.8
- Installation: Anaconda, Spyder, pip
- System: Windows

**Comment by lisphilar** [comment]
Thank you!
I merged your pull request.

**Comment by Inglezos** [comment]
I updated again the code with a new pull request. We can close this issue now.

**Comment by lisphilar** [comment]
Thank you for your nice work!
I merged #311
